src/mjolnir/osmway.cc

```diff
@@ -169,5 +169,22 @@ OSMWay::GetNames(const std::string& ref, const UniqueNames& name_offset_map, uin
   return names;
 }
 
+// Get the tagged names for an edge
+std::vector<std::string> OSMWay::GetTaggedNames(const UniqueNames& name_offset_map) const {
+
+  std::vector<std::string> names;
+  std::vector<std::string> tokens;
+
+  if (tunnel_name_index_ != 0) {
+    // tunnel names
+    tokens = GetTagTokens(name_offset_map.name(tunnel_name_index_));
+    for (const auto& t : tokens) {
+      names.emplace_back(std::to_string(static_cast<uint8_t>(TaggedName::kTunnel)) + t);
```

Comment:
cc @SiarheiFedartsou 

@gknisely @kdiluca this was a big mistake, we werent supposed to to_string the enum value. that limits this pattern to work for only 1-9.... we were supposed to do:

```c++
// we cast the enum type to std::string::CharT, we know its gauranteed to be 8bits wide that first byte fits 256 values
names.emplace_back(std::string(1, TaggedName::kTunnel) + t);
```

ill write up an issue about this...

Reply:
@kevinkreiser ok ty